pkg/plan/recipe/install_plans.go

```diff
@@ -93,6 +107,8 @@ func BuildCRIPlan(criSpec *baremetalspecv1.ContainerRuntime, cfg *envcfg.EnvSpec
 	switch pkgType {
 	case resource.PkgTypeRPM:
 		b.AddResource("install:docker", &resource.RPM{Name: criSpec.Package, Version: criSpec.Version})
+	case resource.PkgTypeRHEL:
```

Comment:
Looks like this can be removed and just add resource.PkgTypeRHEL to the above case statement.